docs/data/hubble/data-catalog/data-model-diagram.mdx

```diff
@@ -3,8 +3,8 @@ title: Data Model Diagram
 sidebar_position: 0
 ---
 
+:::info For more detailed information see the [Full Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-66671e056bc9d447b1501801) or the [Compressed Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-Compressed-666730996bc9d447b151a181). :::
```

Comment:
```suggestion
:::info 

For more detailed information see the [Full Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-66671e056bc9d447b1501801) or the [Compressed Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-Compressed-666730996bc9d447b151a181).

:::
```
This might give some trouble with markdown parsing.

Reply:
Thanks, my vim auto formatted markdowns. Got look into that, updated